source/fuzz/fuzzer_util.h

```diff
@@ -349,9 +349,10 @@ uint32_t MaybeGetStructType(opt::IRContext* ir_context,
                             const std::vector<uint32_t>& component_type_ids);
 
 // Recursive definition is the following:
-// - if |scalar_or_composite_type_id| is a result id of a scalar type - returns
-//   a result id of the following constants (depending on the type): int -> 0,
-//   float -> 0.0, bool -> false.
+// - if |scalar_or_composite_type_id| is a result id of a scalar type -
+// returns
+//   a result id of the following constants (depending on the type): int ->
+//   0, float -> 0.0, bool -> false.
```

Comment:
The diff between the commit before your changes and your 6 commits shows a change in formatting here, including something that's wrong - "returns" is on its own on a line and not indented properly.

It doesn't look to me like there's a need for any change in formatting.  Can you please check?

Reply:
Reverted